packages/validator/src/services/validatorStore.ts

```diff
@@ -233,6 +233,24 @@ export class ValidatorStore {
     return this.validators.get(pubkeyHex)?.graffiti ?? this.defaultProposerConfig.graffiti;
   }
 
+  setGraffiti(pubkeyHex: PubkeyHex, graffiti: string): void {
+    const validatorData = this.validators.get(pubkeyHex);
+    if (validatorData === undefined) {
+      throw Error(`Validator pubkey ${pubkeyHex} not known`);
+    }
+    // This should directly modify data in the map
```

Comment:
this comment doesn't sound very convincing, what about "Mutate data object to set graffiti" or just omit the comment all together, should be a given that devs know how object mutation works in js land